testdata/project-v4/internal/webhook/v1/issuer_webhook.go

```diff
@@ -54,7 +54,7 @@ type IssuerCustomDefaulter struct {
 var _ webhook.CustomDefaulter = &IssuerCustomDefaulter{}
 
 // Default implements webhook.CustomDefaulter so a webhook will be registered for the Kind Issuer.
-func (d *IssuerCustomDefaulter) Default(ctx context.Context, obj runtime.Object) error {
+func (d *IssuerCustomDefaulter) Default(_ context.Context, obj runtime.Object) error {
```

Comment:
Could we keep ctx here?
We do not have the implementation using ctx, but the end user will use it when supplement the implementation for it